models/store/src/store/store-draft/presets/sample-data-b2b/de-fr-uk.ts

```diff
@@ -39,6 +48,7 @@ const deFrUk = (): TStoreDraftBuilder =>
     ])
     .supplyChannels([
       KeyReferenceDraft.presets.channel().key(euWarehouseChannel.key),
-    ]);
+    ])
+    .productSelections([defaultProductSelectionSetting]);
```

Comment:
Does this need to be inside a `faker`? I feel like we've seen that before?

Reply:
how is it different than `supplyChannels`, since it's pre-built?  I can certainly put it in a `fake` block tho

Reply:
thinking about this more, if we wanted to put a random product selection into the default (`.random()`) `StoreDraft` defined in `store-draft/generator.ts`, we would specify that in a faker block.

but I don't believe we need a faker block for a one-off pre-built value in a preset